certbot/certbot/util.py

```diff
@@ -505,12 +516,12 @@ def enforce_le_validity(domain):
                     label, domain))
     return domain
 
-def enforce_domain_sanity(domain):
+def enforce_domain_sanity(domain: str) -> str:
```

Comment:
Should this be `Union[str, bytes]`? The function body appears to be written to accept either.

Reply:
Yes, fixed.